### PR TITLE
Hide checkbox

### PR DESCRIPTION
**Pull request by JayPatel200**
1. Made the checkboxes bigger.
2. Checkboxes now disappear once clicked.
3. Made the text to checkboxes, bold.
4. Removed unwanted functions and added comments for better code understanding.
5. Added description/label to CC
6. Removed conflicts.

Now it looks like,

final checkbox

### CODE REVIEW

File: client/styles/button.module.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    padding: 0 1em;
+    background: #34343E;
+    border-radius: 0.125em;
+    background-clip: padding-box;
+  
+    margin-right: 1em
+}
+  
+.button:before {
+      content: "\2190";
```
Comment (just1ngray) [inline client/styles/button.module.css]: 6 spaces?

File: client/styles/button.module.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (just1ngray) [inline client/styles/button.module.css]: Do you mean to commit this file?

File: client/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "@tippy.js/react": "^3.1.1",
     "axios": "^0.20.0",
     "jsonwebtoken": "^8.5.1",
-    "next": "9.5.3",
+    "next": "^9.5.3",
```
Comment (just1ngray) [inline client/package.json]: Why?

File: client/package-lock.json [hunk 4]
```diff
@@ -5,12 +5,27 @@
   "requires": true,
```
Comment (just1ngray) [inline client/package-lock.json]: We have this file in our gitignore. Why are you committing it?

**Review by just1ngray — CHANGES_REQUESTED** [review]
Functionally this is looking great. For maintainability of our repository, please address my specific comments and then we can approve this!

I tried fixing this myself, but without messing with your commits directly, I'll leave it to you.

File: client/styles/button.module.css (hunk 2 above)
Comment (JayPatel200) [reply client/styles/button.module.css]: Nope, was unintentional

**Comment by JayPatel200** [comment]
Too many conflicts, easier to make a new PR

**Pull request closed without merging**